app/error_handlers.py

```diff
@@ -64,8 +64,8 @@ def path_starts_with(suffix):
             return await too_many_requests_register(request)
         except InactiveCaseError:
             return await inactive_case(request)
-        except ExerciseClosedError as ex:
-            return await ce_closed(request, ex.collection_exercise_id)
+        except ExerciseClosedError:
+            return await survey_closed(request)
```

Comment:
Strictly this is a collection exercise (ce) closed, rather than a survey closed - surveys can have multiple ce's, but never close, so think ce_closed and passing the collection_exercise_id will probably be required

Reply:
Updated.